.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
+    directories:
+      - /
+      - fuzz
+    schedule:
+      interval: weekly
+  - package-ecosystem: docker
+    directory: /
+    schedule:
+      interval: weekly
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+  - package-ecosystem: pip
+    directory: crates/redb-python
+    schedule:
+      interval: weekly
```

Comment:
I don't really like dependabot. Let's remove this. The rest looks fine as long as the tests pass. Although I kind of thing that rocksdb version has a bug

Reply:
Np. That's your choice